HelpSource/Classes/Object.schelp

```diff
@@ -764,10 +755,10 @@ teletype::
 
 ## Some instance objects return more detailed information, such as
```

Comment:
just saw this one: "Some instance objects" -> "Some instances"

Reply:
Thanks, done!